### DISCUSSION TITLE
I can not use useEventListener in Nuxt3

### DISCUSSION THREAD

**Discussion by erdemeren1234** [vueuse/vueuse#2672] [Q&A] [answered]
it send "document is not defined" error. Firstly the code worked but when i refreshed the page, i got this error again and again. I used like in its document. But useMouse() worked.

```
<script lang="ts" setup>
const { x, y } = useMouse()

useEventListener(document, "visibilitychange", (e) => {
  console.log(e); 
});
</script>
```

**Comment by hslee2008** [comment, marked as answer]
Document is not defined because the script loads before the document.

Use the `onMounted()` tool.

```
<script lang="ts" setup>
const { x, y } = useMouse()

onMounted(() => useEventListener(document, "visibilitychange", (e) => {
  console.log(e); 
}))
</script>
```

**Reply by erdemeren1234** [reply]
Thanks! Its documentation is wrong then. The code is using on its documentation as i used above.
